hphp/runtime/vm/jit/abi-arm.cpp

```diff
@@ -89,9 +89,15 @@ const RegSet kScratchCrossTraceRegs =
 /*
  * Helper code ABI registers.
  */
-const RegSet kGPHelperRegs = rAsm | vixl::x14;
+const RegSet kGPHelperRegs = vixl::x14 | rAsm;
 const RegSet kSIMDHelperRegs = vixl::d5 | vixl::d6 | vixl::d7;
 
+/*
+ * Fast Stub ABI registers.
+ */
+const RegSet kGPFastStubRegs = vixl::x16 | rAsm;
```

Comment:
I *think* this should be fine. The VIXL macroassembler uses its temps only within the scope of one [macro-]assembly instruction. The only danger is in crafting a fast stub which is using x16 and then using a vasm instruction which is emitted with a VIXL macroassembly routine that uses x16 as a temp. Preventing this is up to the writer of the fast stub. You think that's a reasonable caveat, or should we try to come up with some more explicit way of preventing this?